### DISCUSSION TITLE
[Support]: Frigate 0.14 in HA. Where is my snapshots saved?

### DISCUSSION THREAD

**Discussion by Erisvaenget27** [blakeblackshear/frigate#13263] [General Support] [answered]
### Describe the problem you are having

Great job in re-doing the UI. Although I seem to struggle finding the snapshot my camera have been taking. In the old UI it was quite easy to monitor and delete when I didnt needed them anymore. My issue now is that my backup of HA is increasing in size.

Thanks!

### Version

0.14

### Frigate config file

```yaml
NA
```

### Relevant log output

```shell
NA
```

### FFprobe output from your camera

```shell
NA
```

### Operating system

HassOS

### Install method

HassOS Addon

### Object Detector

CPU (no coral)

### Network connection

Wired

### Camera make and model

Hikvision

**Comment by hawkeye217** [comment, marked as answer]
See the [FAQ](https://github.com/blakeblackshear/frigate/discussions/12724), Usage question 8.

**Reply by Erisvaenget27** [reply]
Right. 

I found it. The simple answer to the next ones looking. 

Media: 
![image](https://github.com/user-attachments/assets/2c45ed19-1d1b-4628-841d-329afbd83ecc)
